python/catkin/test_results.py

```diff
@@ -134,6 +135,12 @@ def test_results(test_results_dir):
                 print('Skipping "%s": %s' % (name, str(e)))
                 continue
             results[name] = (num_tests, num_errors, num_failures)
+            if show_verbose and (num_errors + num_failures > 0):
+                print("Test %s was failed" % (name))
+                print("-------------------------------------------------")
+                verbose = open(filename_abs, "r").read()
```

Comment:
Have you tried this with a test result file which contains non-ascii characters?